src/components/shared/articleBody.tsx

```diff
@@ -36,16 +33,8 @@ const ArticleBodyStyles = (pillarStyles: PillarStyles): SerializedStyles => css`
         border: none;
     }
 
-    ${until.wide} {
-        figure:not(.interactive) {
-            margin-left: ${basePx(-1)};
-            margin-right: ${basePx(-1)};
-        }
-    }
-
     ${adStyles}
     ${sidePadding}
-    ${commonArticleStyles(pillarStyles)}
```

Comment:
Guessing once liveblogs are migrated this function can be deleted?

Reply:
The `commonArticleStyles` function? Yes I think so.